src/webgpu/api/operation/queue/writeBuffer.spec.ts

```diff
@@ -8,3 +8,89 @@ import { makeTestGroup } from '../../../../common/framework/test_group.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export const g = makeTestGroup(GPUTest);
+
+g.test('writeBuffer,Ranges')
```

Comment:
`writeBuffer` can be omitted here because it's redundant with the filename. So just `'ranges'` or maybe `typed_array_types`